articles/azure-functions/functions-bindings-kusto-managed-identity.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Connect a function app to Kusto with managed identity - Preview
+description: Use mnagaed identity in Kusto bindings
+author: ramacg
+ms.topic: reference
+ms.date: 03/27/2023
+ms.author: ramacg
+ms.reviewer: 
+zone_pivot_groups: programming-languages-set-functions-lang-workers
+---
+# Tutorial: Connect a function app to Kusto with managed identity

```

Comment:
# Tutorial: Connect an Azure Functions app to Azure Data Explorer with managed identity